swagger.yml

```diff
@@ -628,6 +628,26 @@ paths:
             $ref: '#/definitions/deploy'
         default:
           $ref: '#/responses/error'
+  /sites/{site_id}/deploys/{deploy_id}/cancel:
```

Comment:
Late to the party, but URL for this is `/deploys/{deploy_id}/cancel`, no need for the `site_id` prefix.